Python/thread_pthread.h

```diff
@@ -81,6 +81,8 @@
    we need to add 0 to make it work there as well. */
 #if (_POSIX_SEMAPHORES+0) == -1
 #define HAVE_BROKEN_POSIX_SEMAPHORES
+#elif defined(__NetBSD__)
+#define HAVE_BROKEN_POSIX_SEMAPHORES
```

Comment:
``configure.ac`` has a block to define ``HAVE_BROKEN_POSIX_SEMAPHORES`` for SunOS and AIX. Please move the NetBSD code there, too.

Reply:
I've changed this as requested.
I noticed that the generated files from autoconf are in the repository, but I have a newer version installed, so I didn't include these files in the commit.